src/mam/mod_mam_muc_cassandra_arch.erl

```diff
@@ -771,6 +771,6 @@ stored_binary_to_packet(HostType, Bin) ->
 db_message_format(HostType) ->
     gen_mod:get_module_opt(HostType, ?MODULE, db_message_format).
 
--spec pool_name(HostType :: host_type()) -> mongoose_wpool:pool_name().
+-spec pool_name(HostType :: host_type()) -> default. %% returns mongoose_wpool:pool_name().
```

Comment:
why, isn't default a correct `mongoose_wpool:pool_name()`?

Reply:
it's correct pool_name, except dialyzer will say the real type is `default`, so go and fix it.